### PR TITLE
Update Joven About us

### LINKED ISSUE #5: Update Joven AboutUs
Add in your own profie photo and update your link

### PR DESCRIPTION
**Pull request by Joven-Heng**
Close #5

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -5,55 +5,35 @@ title: About Us
...
-
 ## Project team
 
-### John Doe
+### Fong Wei Jie
 
 <img src="images/johndoe.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
```
Comment (WeiJie96) [inline docs/AboutUs.md:17]: Perhaps we could keep keep this so we can change it individually

File: docs/AboutUs.md [hunk 2]
```diff
@@ -5,55 +5,35 @@ title: About Us
...
-[[portfolio](team/johndoe.md)]
```
Comment (WeiJie96) [inline docs/AboutUs.md:18]: Perhaps we could keep keep this so we can change it in future

File: docs/AboutUs.md [hunk 3]
```diff
@@ -5,55 +5,35 @@ title: About Us
...
-* Responsibilities: Dev Ops + Threading
+<img src="images/joven-heng.png" width="200px">
 
-### James Doe
+[Joven github](http://github.com/Joven-Heng) 
 
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
```
Comment (WeiJie96) [inline docs/AboutUs.md:56]: Perhaps we could keep keep this so we can change it in future

**Review by WeiJie96 — CHANGES_REQUESTED** [review]
Overall is good, but should keep 3 lines.

File: docs/AboutUs.md (hunk 3 above)
Comment (Joven-Heng) [reply docs/AboutUs.md:56]: Ok fixed those, have a look

**Review by WeiJie96 — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -19,5 +19,5 @@ such that it can be used as a client tracking software for insurance agents.
 * For the detailed documentation of this project, see [here](https://ay2021s1-cs2103-t16-2.github.io/tp/).
 
 * Credits:
-* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
-* Professor Damith, Teaching Assistant Kim Saehyun and fellow peer reviewers who have graciously tested our app :)
+    * This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```
Comment (ijavierja) [inline README.md]: Is there any reason for the space infront?

**Review by ijavierja — COMMENTED** [review]
LGTM

**Pull request merged**